src/BenchmarkDotNet/Helpers/SourceCodeHelper.cs

```diff
@@ -18,9 +18,9 @@ public static string ToSourceCode(object value)
                 case bool b:
                     return b.ToLowerCase();
                 case string text:
-                    return $"$@\"{text.Replace("\"", "\"\"").Replace("{", "{{").Replace("}", "}}")}\"";
+                    return "$" + text.EscapeSpecialCharacters(true).Replace("{", "{{").Replace("}", "}}");
```

Comment:
But most likely we need it. I don't remember exactly why, but I am guessing that users can use paths for `Params` and by using `@` we can get all the C# goodness that it provides (we need less escaping)

```suggestion
                    return "$@" + text.EscapeSpecialCharacters(true).Replace("{", "{{").Replace("}", "}}");
```

Reply:
>It will escape a string 2 times: "\\n" -> @"\\n" -> "\\\n" 

```C#
text = "\n" // special char
text.EscapeSpecialCharacters(true) -> "\\n"
@ + text -> @"\\n" -> "\\\n"
```